### DISCUSSION TITLE
未ログイン画面からも、「Markdown記法チートシート」へジャンプしたい

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#315] [要望・アイデア・不具合]
「Markdown記法チートシート」は頻繁に便利に参照させていただいております。しかし、Qiitaにログインしてから、投稿エディタ画面を開いて、その上の方にある「？」マークをクリックするとやっと見られるという深窓の文書です。

　私は、ログインしない状態で Qiita を開き、色々な記事を参照しながら、汎用エディタで投稿予備原稿を作ることが良くあります。チートシートには Qiita のホーム画面から直接リンクしていただけると大助かりです。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
「Markdown記法チートシート」をご活用いただきありがとうございます。
恐れ入りますが、他の情報が探しにくくなってしまうため、ホーム画面に「Markdown記法チートシート」を表示することは難しいです。
ブラウザで記事をブックマークしていただいたり、以下ヘルプページもご活用いただければ幸いです🙇
https://help.qiita.com/ja/articles/qiita-markdown

**Reply by tsubolabo** [reply]
ご回答ありがとうございます。

>　
>ブラウザで記事をブックマークしていただいたり、
>　

それもそうですね。そうしました。Qiitaに甘え過ぎていました。すみません。

**Reply by tsubolabo** [reply]
なぜ、「こんなバカな提案をしてしまったんだろう」という反省の中で、ここに至った経緯を整理してみました。

+ 「Markdown記法チートシート」はパスワードを入力してログインしなければ見せて貰えない深窓の文書である（勘違い）。
　↓
+ 未ログイン状態でチートシートのURLを直接たたいても、システムに弾かれてしまうに違いない（勘違い）。
　↓
+ ぜひ、万人に一般公開して貰いたい。
　↓
+ Discussions に「要望・アイデア・不具合」で提案。
　↓
+ Qiita から回答「ブラウザで記事をブックマークしていただいたり、」
　↓
+ 自分の思い「そんなことができるのなら、最初から提案などしていないよ」
　↓
+ Discussion #302 関連の事例調査のために、チートシートを隅から隅まで読んでみた。
　↓
+ 深窓の文書だと思っていたチートシートが、一般会員の投稿記事と同列の一文書に過ぎないことを始めて認識。
　↓
+ それなら「ログインしなくても読めるんだ」と気づく。
　↓
+ 直接URLをたたいてやってみた。「お～っ」（←今ここ）。

　ここで思うのは次のようなことです。
「こんなに頻繁に使われるQiitaの公式文書が、なぜこのような低位の位置づけになっており、しかも、直接リンクまでの手順をややこしくしているのだろうか。」

　時々、チートシートのつまみ食い的な内容で、投稿者本人が個人的に参照するだけのような記事が投稿され、shiracamus さんから叱られているのを目にします。このような投稿があるのも、オリジナルのチートシートへの直接リンクがややこしいからではないでしょうか？

**Reply by shiracamus** [reply]
叱っているつもりはないですが・・・
ページ一番下の「ガイドとヘルプ」に「マークダウンの書き方」のリンクがあるといいかもしれませんね。
「ヘルプ」の中の「Qiita Markdown」や「Qiitaの投稿をMarkdown表示にする方法」の中にリンクがありました。
https://help.qiita.com/
「記事の投稿」の最初の項目として「マークダウンを使った記事の書き方」と分かりやすく書いておくと良さそうですね。
